doc/en/index.rst

```diff
@@ -17,7 +17,7 @@ The ``pytest`` framework makes it easy to write small, readable tests, and can
 scale to support complex functional testing for applications and libraries.
 
 
-**Pythons**: ``pytest`` requires: Python 3.6, 3.7, 3.8, 3.9, or PyPy3.
+``pytest`` requires: Python 3.7+ and PyPy3.
```

Comment:
I think `or` makes more sense here (and below)

Reply:
I also changed `Python 3.6+ and PyPy3` to `Python 3.6+ or PyPy3` in `Features` section of README.
Does it also make sence?